openmdao/components/tests/test_demux_comp.py

```diff
@@ -181,7 +180,9 @@ def test(self):
 
         p.run_model()
 
-        expected = np.arctan(p['pos_ecef'][:, 1] / p['pos_ecef'][:, 0])
+        p.model.connect('indep.x', 'C1.x', src_indices=om.slicer[:, 1])
```

Comment:
What is a connect statement doing after the run_model?

Reply:
Fixed. 
Thanks for catching that.